app/__init__.py

```diff
@@ -18,6 +19,17 @@
 content_loader = ContentLoader('app/content')
 
 
+def _try_load_manifest(application, data, question_set, manifest):
+    try:
+        content_loader.load_manifest(data['slug'], question_set, manifest)
+
+    except ContentNotFoundError:
+        # Not all frameworks have this, so no need to panic (e.g. G-Cloud 4, G-Cloud 5, G-Cloud-6)
```

Comment:
This comment doesn't make any sense when this functionality is abstracted into it's own function. If anything, this comment should be made when we decide to use `_try_load_manifest` as you need the context (it doesn't apply for every time we run the helper func).